examples/tracetest-opensearch/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: '3'
+services:
+
+    tracetest:
+        container_name: tracetest
+        image: kubeshop/tracetest:v0.6.4
+        volumes:
+            - type: bind
+              source: ./tracetest-config.yaml
+              target: /app/config.yaml
+        ports:
+            - 8080:8080
+        depends_on:
+            postgres:
+                condition: service_healthy
+            otel-collector:
+                condition: service_started
+
+    postgres:
+        container_name: postgres
```

Comment:
This setting is generating issues with a preexisting container: 
![Screen Shot 2022-08-23 at 12 31 36](https://user-images.githubusercontent.com/314548/186199753-4f0b79b8-4402-49e4-834b-ef1cd87ac137.png)

Reply:
Removed all `container_name` attributes from the docker-compose file